snuba/query/allocation_policies/__init__.py

```diff
@@ -173,9 +180,9 @@ class AllocationPolicy(ABC, metaclass=RegisteredClass):
     * get_quota_allowance will throw an AllocationPolicyViolation if _get_quota_allowance().can_run is false.
         this is to keep with the pattern in `db_query.py` which communicates error states with exceptions. There is no other
         reason. For more information see snuba.web.db_query.db_query
-    * Every allocation policy takes a `storage_set_key` in its init. The storage_set_key is like a pseudo-tenant. In different
+    * Every allocation policy takes a `storage_key` in its init. The storage_key is like a pseudo-tenant. In different
         environments, storages may be co-located on the same cluster. To facilitate resource sharing, every allocation policy
-        knows which storage_set_key it is serving. This is currently not used
+        knows which storage_key it is serving. This is currently not used
```

Comment:
Seems like the storage key is actually used now and it's pretty important to every policy's operation